moveit_experimental/moveit_jog_arm/config/ur_simulated_config.yaml

```diff
@@ -53,4 +53,5 @@ command_out_topic: joint_group_position_controller/command # Publish outgoing co
 ## Collision checking for the entire robot body
 check_collisions: true # Check collisions?
 collision_check_rate: 10 # [Hz] Collision-checking can easily bog down a CPU if done too often.
-collision_proximity_threshold: 0.01 # Start decelerating when a collision is this far [m]
+self_collision_proximity_threshold: 0.01 # Start decelerating when a self-collision is this far [m]
+scene_collision_proximity_threshold: 0.02 # Start decelerating when a scene collision is this far [m]
```

Comment:
Maybe we should increase the default scene proximity threshold a bit?

Reply:
I bumped it to 5cm.